consensus/clique/snapshot.go

```diff
@@ -22,12 +22,13 @@ import (
 	"sort"
 	"time"
 
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
The import order is whatever `goimports` gives us. I prefer that vs manual groups.

Reply:
the new import order is given by `goimports`. you can config `goimports` with `-local github.com/ethereum/go-ethereum`, that will give you a proper order.

Reply:
Our policy is that we use what `goimports` does by default. We don't want to change all code to have three import 'groups'.